app/controllers/admin/custom_fields/hierarchy/items_controller.rb

```diff
@@ -53,13 +53,42 @@ def new
           respond_with_turbo_streams
         end
 
+        def edit
+          update_via_turbo_stream(
+            component: ItemComponent.new(
+              custom_field: @custom_field,
+              hierarchy_item: @hierarchy_item,
+              edit_item_form_data: { show: true }
+            )
+          )
+
+          respond_with_turbo_streams
+        end
+
         def create
           ::CustomFields::Hierarchy::HierarchicalItemService
             .new
             .insert_item(**item_input)
             .either(
               ->(_) { update_via_turbo_stream(component: ItemsComponent.new(custom_field: @custom_field)) },
-              ->(validation_result) { add_errors_to_form(validation_result) }
+              ->(validation_result) { add_errors_to_new_form(validation_result) }
+            )
+
+          respond_with_turbo_streams
+        end
+
+        def update
+          ::CustomFields::Hierarchy::HierarchicalItemService
+            .new
+            .update_item(item: @hierarchy_item, label: item_input[:label], short: item_input[:short])
+            .either(
+              ->(_) {
```

Comment:
🟡 if we have multi-line blocks, IMHO we should use `do end`, not curlies

Reply:
Agreed. I usually rely on rubocop magic to complain about these. But this is an easy and happy fix :)

Reply:
@Kharonus you know that this can change semantics too? Not in this case but overall.